#include <iostream>

using namespace std;

int main()
{
    try {
        testExceptionsInMain();
    }
    catch ()
    {

    }
}
